.changes/nextrelease/lambda_tcp_keepalive.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "enhancement",
```

Comment:
Since this is changing a transport level option, I'd recommend adding setting this as a `"feature"` level to get a minor version bump.